### PR TITLE
Add new 'influx telegrafs` commands

### LINKED ISSUE #1226: Add 'influx telegrafs' command and subcommands
PR: https://github.com/influxdata/influxdb/pull/18997

The `influx` CLI now has an `influx telegrafs` command used for managing Telegraf configurations. We need to add the following:

- [x] Reference docs for each of the new CLI commands

##### Relevant URLs
- https://v2.docs.influxdata.com/v2.0/reference/cli/influx/

### PR DESCRIPTION
**Pull request by sanderson**
Closes #1226 

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/reference/cli/influx/telegrafs/_index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: influx telegrafs
+description: >
+ placeholder
```
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/telegrafs/_index.md:4]: to be sure, are these still placeholders?

File: content/v2.0/reference/cli/influx/telegrafs/rm.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#### Aliases
+`rm`, `remove`
+
+## Flags
+| Flag |                  | Description                         | Input type  | {{< cli/mapped >}}     |
+|:---- |:---              |:-----------                         |:----------: |:------------------     |
+| `-h` | `--help`         | Help for the `rm` command           |             |                        |
+|      | `--hide-headers` | Hide the table headers              |             | `$INFLUX_HIDE_HEADERS` |
+| `-i` | `--id`           | Telegraf configuration ID to remove | strings     |                        |
+|      | `--json`         | Output data as json                 |             | `$INFLUX_OUTPUT_JSON`  |
```
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/telegrafs/rm.md:28]: Curious about when you'd want to output json if you're trying to remove configuration?

File: content/v2.0/reference/cli/influx/telegrafs/update.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+ placeholder
+menu:
+  v2_0_ref:
+    name: influx telegrafs update
+    parent: influx telegrafs
+weight: 201
+---
+
+The `influx telegrafs update` command updates a Telegraf configuration to match the specified parameters.
+If a name or description are not provided, they are set to an empty string.
```
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/telegrafs/update.md:13]: Interesting, so the existing name or description wouldn't be retained?

File: content/v2.0/reference/cli/influx/telegrafs/_index.md (hunk 1 above)
Comment (sanderson) [reply content/v2.0/reference/cli/influx/telegrafs/_index.md:4]: Doh, nice catch. I'll update these.

File: content/v2.0/reference/cli/influx/telegrafs/rm.md (hunk 2 above)
Comment (sanderson) [reply content/v2.0/reference/cli/influx/telegrafs/rm.md:28]: This is a standard implementation across the entire CLI. If a user is scripting the CLI, JSON output is much easier to parse than the standard output.

File: content/v2.0/reference/cli/influx/telegrafs/update.md (hunk 3 above)
Comment (sanderson) [reply content/v2.0/reference/cli/influx/telegrafs/update.md:13]: Currently, no, they wouldn't (as far as I understand) @jsteenb2 👆 ?

**Pull request merged**